tests/monster_test_generated.rs

```diff
@@ -1357,8 +1425,12 @@ impl<'a> Monster<'a> {
     self._tab.get::<flatbuffers::ForwardsUOffset<flatbuffers::Vector<'a, Color>>>(Monster::VT_VECTOR_OF_ENUMS, None)
   }
   #[inline]
-  pub fn signed_enum(&self) -> Race {
-    self._tab.get::<Race>(Monster::VT_SIGNED_ENUM, Some(Race::None)).unwrap()
+  pub fn signed_enum(&self) -> Result<Race, flatbuffers::ConvertError<i8>> {
+    self._tab.get::<i8>(Monster::VT_SIGNED_ENUM, Some(-1)).map(|value| value.try_into()).unwrap()
+  }
+  #[inline]
+  pub unsafe fn signed_enum_unchecked(&self) -> Race {
```

Comment:
There should be an method for getting the raw `i8` value from `signed_enum` directly.

Reply:
I'll add reverse TryFrom.

Reply:
> Or rather From, since it's infallible.

No, not a `Enum as i8`, but instead a safe infallible getter that gets `i8` from the struct.

Reply:
sure